### ISSUE TITLE
Leftover PHP Code in ToolDocs

### ISSUE THREAD

**Issue by gbrandt6** [broadinstitute/gatk#7024]
### Description 
This involves the Tool Docs pages. When sharing the link to a tool docs page, the link description shows the PHP code from the old website. 

For example, 
**SelectVariants**
_include '../../../../common/include/common.php'; include_once '../../../config.php'; $module = modules::GATK; $name = docSN::toolDocs; printHeader($module, $name, topSN::guide); ..._

This PHP code does not appear in the actual Tool Docs, so there is no visible problem on the website.

**Comment by droazen** [comment]
@cmnbroad Any insight into what might be causing this one? Is this a problem with our documentation templates?

**Comment by cmnbroad** [comment]
@gbrandt6 Can you be more specific about how you're creating the link and where you see this text.

**Comment by gbrandt6** [comment]
@cmnbroad yes, it is visible when sharing a link through Slack. It might be in other places as well, but that is where I have seen it. If you send one of the Tool Docs pages in a channel or conversation, the text is visible in the preview/link description.

**Comment by cmnbroad** [comment]
Got it thanks. I'll take a look.

**Comment by cmnbroad** [comment]
@gbrandt6 The online docs have a `meta` tag with a `description` attribute that contains the php code:

`<meta name="description" content="include '../../../../common/include/common.php';     include_once '../../../config.php';     $module = modules::GATK;     $name =..." /><meta property="og:image" content="https://theme.zdassets.com/theme_assets/2378360/ceb967563bfc35b9ab52ba13c0f5c4d870dff930.png" />
`

but that `meta` tag isn't generated by the doc build process (the raw doc files produced by the build have no `meta` tags). So these must be introduced by whatever transformation is done as part of publishing.

The raw templates used by the build do contain a `php` tag that contains the php code that winds up in the meta tag. It may be that the solution involves adding meta tags to the raw templates used by the build, but determining what changes need to be made requires someone with knowledge of the publish process. Ideally we would nominate someone from comms who understands that process, and have them also be the steward of the templates used by the build (I'm happy to give a tutorial on the doc build if that helps). It would also help with other questions such as whether we even need that php code in the templates...

**Comment by vdauwera** [comment]
The php code should no longer be necessary, we used it as part of the page structure on the old website, but the new website uses a different system. So I believe this is legacy crud that can be removed. Just need to check the github->zendesk push system to make sure the crud isn't somehow important to the process. Happy to consult.

**Comment by droazen** [comment]
@gbrandt6 Since this is not a GATK issue, but a problem in the github -> zendesk push system external to GATK, can this be closed?

**Comment by vdauwera** [comment]
@droazen I believe the raw templates containing the php code (which should be removed) are in the GATK repo so this can't be closed quite yet.

**Comment by gbrandt6** [comment]
We were looking into where the php code is coming from and it looks like it is in the template files when the docs are built by gradle. @vdauwera found the code in the generic.template.html. It looks like we should be able to solve the issue without changing every single tool doc.

![](https://files.slack.com/files-pri/T0166C33N6A-F01LESQPQ8L/screen_shot_2021-01-28_at_4.26.20_pm.png)
